server/Cargo.toml

```diff
@@ -50,4 +50,17 @@ static-files = "0.2.1"
 walkdir = "2"
 
 [build-dependencies]
-static-files = "0.2.1"
\ No newline at end of file
+static-files = "0.2.1"
+cargo_toml = "0.11.5"
+ureq = "2.5.0"
+sha1_smol = {version = "1.0.0", features=["std"]}
+hex = "0.4.3"
+zip = { git = "https://github.com/zip-rs/zip" }
+
+[package.metadata.parseable_ui]
+assets-url = "http://localhost:5555/ui/build.zip"
+assets-sha1 = "541f1887087ff80149b1d19bce802b4e359cb2e3"
+# specifies where to find local assests
```

Comment:
Published a build.zip on the frontend repo. 

https://github.com/parseablehq/frontend/releases/tag/v0.0.1

Sha is `d9243bde7a98bdd8a19ed47cca2f12f9a5ce40ba`. Can you please change the links here to the correct links and try building once?

Reply:
Yeah, I'll update it right away